catalyst-gateway/bin/src/service/common/types/cardano/nonce.rs

```diff
@@ -0,0 +1,126 @@
+//! Nonce
+
+use std::sync::LazyLock;
+
+use poem_openapi::{
+    registry::{MetaSchema, MetaSchemaRef},
+    types::{Example, ParseError, ParseFromJSON, ParseFromParameter, ParseResult, ToJSON, Type},
+};
+use serde_json::Value;
+
+use super::slot_no;
+
+/// Title.
+const TITLE: &str = "Nonce";
+/// Description.
+const DESCRIPTION: &str = "The current slot at the time a transaction was posted.
+Used to ensure out of order inclusion on-chain can be detected.
+
+*Note: Because a Nonce should never be greater than the slot of the transaction it is found in, 
+excessively large nonces are capped to the transactions slot number.*";
```

Comment:
```suggestion
const DESCRIPTION: &str = "A number used once that must increase sequentially for
all transactions associated with the same staking key. 
It is recommended to use current slot number as a nonce.
Used to ensure out of order inclusion on-chain can be detected.

*Note: Because a Nonce should never be greater than the slot of the transaction it is found in, 
excessively large nonces are capped to the transactions slot number.*";
```

Reply:
Nope, because the wording from the CIP is too vague and open.
The comment I have is accurate.
The CIP suggests using the slot# and then lets it be anything, which causes issues.  For us, Nonce now means the suggestion, or capped.  Which prevents the issue.